internal/handlers/common/aggregations/stages/collstats.go

```diff
@@ -70,14 +70,18 @@ func newCollStats(stage *types.Document) (aggregations.Stage, error) {
 
 		storageStatsFields := must.NotFail(fields.Get("storageStats")).(*types.Document)
 
+		cs.storageStats.scale = 1
```

Comment:
Adding test revealed that it was set zero.

Reply:
Never mind, it will be handled in https://github.com/FerretDB/FerretDB/pull/3565 PR to avoid conflict